source/reference/api/start.txt

```diff
@@ -202,6 +212,34 @@ Request Body Parameters
 
        .. versionadded:: 1.1
 
+   * - ``verification``
+     - Document 
+     - Optional
+     - Configures the embedded verifier.
+
+       For more information, see :ref:`c2c-start-verifier`.
+
+       .. versionadded:: 1.9
+
+   * - ``verification.enabled``
+     - Bool
+     - Optional
+     - Enables the embedded verifier. The verifier is disabled
+       by default. If the verifier finds no errors,
+       ``mongosync`` transitions to the ``COMMITTED`` state. If
+       it does encounter errors, it fails the migration.
+
+       .. include:: /includes/fact-verifier-memory
+
+       The ``/start`` endpoint returns an error when
+       reversible sync is started with the verifier.
```

Comment:
We allow reverse with the embedded verifier now. But it will only verify the forward direction. It will not verify the reverse or any subsequent reversals.

Reply:
Fixed